front/components/assistant/conversation/lib.ts

```diff
@@ -229,6 +231,34 @@ export async function createConversationWithMessage({
 }): Promise<Result<ConversationType, ConversationErrorType>> {
   const { input, mentions, contentFragments } = messageData;
 
+  const validatedContentFragments: {
+    content: string;
+    title: string;
+    url: null;
+    contentType: ContentFragmentContentType;
+    context: { profilePictureUrl: string | null };
+  }[] = [];
+
+  for (const cf of contentFragments) {
+    if (isSupportedContentFormat(cf.file.type)) {
```

Comment:
This seems to be a bit late, we already created the content fragment.

Reply:
@flvndvd I would personally modify `ContentFragmentInput` to add an extra field `type` or something. This way we would ensure we actually support the type when creating a `ContentFragmentInput`. wdyt?

Reply:
Seen IRL, you can create a function `uploadContentFragments` that returns the right type.